core/domain/skill_services.py

```diff
@@ -754,10 +754,16 @@ def update_skill(committer_id, skill_id, change_list, commit_message):
         for change in change_list
     ])
     if misconception_is_deleted:
+        deleted_skill_misconception_ids = [
+            '%s-%d' % (skill.id, change.misconception_id)
```

Comment:
We have a domain function to generate this, right? Can we use that once?

Reply:
Done.